theme-data/dark/dark-common.json

```diff
@@ -107,7 +107,8 @@
     },
     "button": {
       "border": {
-        "normal": "none"
```

Comment:
`none` was adding a gray, 20% transparency border.

Reply:
The reason for this is that "none" is a keyword in the theme generator and it takes the colour of the background of the object (for example, a button) and places it on the border.